episodes/quantify-transmissibility.Rmd

```diff
@@ -0,0 +1,416 @@
+
+::::::::::::::::::::::::::::::::::::: callout
+### Delays and data
+The number of delays and type of delay is a flexible input that depends on the data. The examples below highlight how the delays can be specified for different data sources:
+
+<center>
+
+| Data source        | Delay(s) |
+| ------------- |-------------|
+|Time of case report      |Incubation period + time from symptom onset to case notification |
+|Time of hospitalisation   |Incubation period + time from symptom onset to hospitalisation     |
+|Time of symptom onset      |Incubation period |
+
+</center>
+
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+
+#### Incubation period distribution 
+
+The distribution of incubation period can usually be obtained from the literature. The package `{epiparameter}` contains a library of epidemiological parameters for different diseases obtained from the literature. 
+
+We specify a (fixed) gamma distribution with mean $\mu = 4$ and standard deviation $\sigma^2= 2$ (shape = $4$, scale = $1$) using the function `dist_spec()` as follows:
+
+```{r}
+incubation_period_fixed <- dist_spec(
+  mean = 4, sd = 2,
+  max = 20, distribution = "gamma"
+)
+incubation_period_fixed
+```
+
+The argument `max` is the maximum value the distribution can take, in this example 20 days. 
+
+::::::::::::::::::::::::::::::::::::: callout
+### Why a gamma distrubution? 
+
+The incubation period has to be positive in value. Therefore we must specific a distribution in `dist_spec` which is for positive values only. 
+
+`dist_spec()` supports log normal and gamma distributions, which are distributions for positive values only. 
+
+For all types of delay, we will need to use distributions for positive values only - we don't want to include delays of negative days in our analysis!
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+
+####  Including distribution uncertainty
+
+To specify a **variable** distribution, we include uncertainty around the mean $\mu$ and standard deviation $\sigma^2$ of our gamma distribution. If our incubation period distribution has a mean $\mu$ and standard deviation $\sigma^2$, then we assume the mean ($\mu$) follows a Normal distribution with standard deviation $\sigma_{\mu}^2$:
+
+$$\mbox{Normal}(\mu,\sigma_{\mu}^2)$$
+
+and a standard deviation ($\sigma^2$) follows a Normal distribution with standard deviation $\sigma_{\sigma^2}^2$:
+
+$$\mbox{Normal}(\sigma^2,\sigma_{\sigma^2}^2).$$
+
+We specify this using `dist_spec` with the additional arguments `mean_sd` ($\sigma_{\mu}^2$) and `sd_sd` ($\sigma_{\sigma^2}^2$).
```

Comment:
Use the [Normal distribution notation](https://en.wikipedia.org/wiki/Normal_distribution#Notation) in this episode.

```suggestion
To specify a **variable** distribution, we include uncertainty around the mean $\mu$ and standard deviation $\sigma$ of our gamma distribution. If our incubation period distribution has a mean $\mu$ and standard deviation $\sigma$, then we assume the mean ($\mu$) follows a Normal distribution with standard deviation $\sigma_{\mu}$:

$$\mbox{Normal}(\mu,\sigma_{\mu}^2)$$

and a standard deviation ($\sigma$) follows a Normal distribution with standard deviation $\sigma_{\sigma}$:

$$\mbox{Normal}(\sigma,\sigma_{\sigma}^2).$$

We specify this using `dist_spec` with the additional arguments `mean_sd` ($\sigma_{\mu}$) and `sd_sd` ($\sigma_{\sigma}$).
```